libraries/AP_Airspeed/AP_Airspeed.h

```diff
@@ -206,15 +300,36 @@ class AP_Airspeed
         Airspeed_Calibration calibration;
         float last_saved_ratio;
         uint8_t counter;
+
+        // Blending error (weight of sensor in blending)
+        float blending_factor;
     } state[AIRSPEED_MAX_SENSORS];
 
+    // Blended airspeed
+    AP_Int8 blend_conf;
+
+    float raw_airspeed_blend;
+    float airspeed_blend;
+    float last_pressure_blend;
+    float filtered_pressure_blend;
+    float corrected_pressure_blend;
+    float EAS2TAS_blend;
+    float temperature_blend;
+    float ratio_blend;
+    float offset_blend;
```

Comment:
please move all these new variables into a struct named "blend" or "blend_state" and remove the _blend on all the names

Reply:
OK